components/Carousel/Carousel.js

```diff
@@ -1,5 +1,53 @@
+class CarouselItem {
+  constructor(element) {
+    this.element = element;
+  }
+
```

Comment:
If you are going to add spacing after the constructor be sure to be consistent across the project. For example you didn't add a space after the end of the select method. It is good to be consistent with these kind of things.